scene/gui/texture_progress_bar.cpp

```diff
@@ -222,40 +222,84 @@ void TextureProgressBar::draw_nine_patch_stretched(const Ref<Texture2D> &p_textu
 		double first_section_size = 0.0;
 		double last_section_size = 0.0;
 		switch (mode) {
-			case FILL_LEFT_TO_RIGHT:
-			case FILL_RIGHT_TO_LEFT: {
+			case FILL_LEFT_TO_RIGHT: {
 				width_total = dst_rect.size.x;
 				width_texture = texture_size.x;
 				first_section_size = topleft.x;
 				last_section_size = bottomright.x;
 			} break;
-			case FILL_TOP_TO_BOTTOM:
-			case FILL_BOTTOM_TO_TOP: {
+			case FILL_RIGHT_TO_LEFT: {
+				width_total = dst_rect.size.x;
+				width_texture = texture_size.x;
+				// For RIGHT_TO_LEFT, first_section_size and last_section_size should switch position
```

Comment:
We now end comments with a period:

```suggestion
				// For RIGHT_TO_LEFT, first_section_size and last_section_size should switch position.
```